docs/debugConfig.md

```diff
@@ -0,0 +1,22 @@
+# Breaking changes to JavaScript Debug Configuration
+
+The debug configuration for v2 of the Azure Functions runtime [has changed](https://github.com/Azure/azure-functions-core-tools/issues/521) and old configurations will likely fail with the error "Cannot connect to runtime process, timeout after 10000 ms". You will automatically be prompted to update your configuration when you open a project. However, you can manually update your project with one of the following options:
+
+The first option is to add the following to your `runFunctionsHost` task in `.vscode\tasks.json`:
+
+```json
```

Comment:
It's really helpful to see the entire task (even if just an example).

Reply:
done